pom.xml

```diff
@@ -354,7 +368,12 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>-Xmx512M</argLine>
+          <!--
+             Use syntax `@{argLine}` for late property evaluation, which
+             ensures that JaCoCo runtime agent can be passed as a VM
+             argument to the application under test.
+           -->
+          <argLine>@{argLine} -Xmx512M</argLine>
```

Comment:
Beware that using @{argLine} already caused troubles on the platform, see [comment on NXBT-1486](https://jira.nuxeo.com/browse/NXBT-1486?focusedCommentId=303324&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-303324).
I strongly suggest to use a dedicated sonar profile that activates analysis: https://github.com/nuxeo/nuxeo/blob/master/pom.xml#L6691

Reply:
:+1: this has to be isolated in a profile in order not to break IDEA unit test execution